core/docker/Dockerfile

```diff
@@ -1,3 +1,4 @@
+# syntax=docker/dockerfile:1-labs
```

Comment:
nit: separate commit

Reply:
No, it's part of that one, --exclude requires it

Reply:
really... Looks like comment for vim syntax coloring.
Awful.